express/api/file-utils.js

```diff
@@ -248,4 +247,95 @@ async function countProjectFiles(project){
   const dirContents = await fs.promises.readdir(`${appConfig.projectDirectory}/${project}`);
   return dirContents.length -1; // ignore the config directory
 }
-module.exports.countProjectFiles = countProjectFiles;
\ No newline at end of file
+module.exports.countProjectFiles = countProjectFiles;
+
+/**
+ * Perform the copying of 'value' and 'include' fields from an original config object into a new one.
+ * 
+ * @param {object} target original config 
+ * @param {object} source new config
+ */
+function copyUserInputValues(target, source) {
+  for (const [key, val] of Object.entries(target)) {
+    if (typeof val === 'object') {
+      if (key === 'VISULATE_VERSION') {
+        continue;
+      }
+      copyUserInputValues(val, source[key]);
+    } else if (key === 'value' || key === 'include') {
+      target[key] = source[key];
+    }
+  }
+}
+
+/**
+ * Merge existing user-inputed values into a new config object created from the 
+ * default template then save the new config object
+ * 
+ * @param {string} projectName    name of the project
+ * @param {object} originalConfig to copy user-inputed values from
+ * @param {object} newConfig      to copy user-inputed values to
+ */
+async function updateConfigObject(projectName, originalConfig, newConfig) {
+  // Rename original for backup
+  await fs.promises.rename(`${appConfig.projectDirectory}/${projectName}/config/ora2pg-conf.json.enc`,
+  `${appConfig.projectDirectory}/${projectName}/config/ora2pg-conf.json.enc.old`);
+  // Copy values from original over to new config
+  copyUserInputValues(newConfig, originalConfig);
+  await saveConfigJson(projectName, newConfig);
+} 
+
+/**
+ * Compare version strings to determine if oldVersion is older, newer, or the same as newVersion.
+ * 
+ * @param {string} oldVersion the version in the user's project config
+ * @param {string} newVersion the version in our default config template
+ * @returns 1 if oldVersion > newVersion, -1 if oldVersion < newVersion, 0 if they are equal
+ */
+function compareVersionStrings(oldVersion, newVersion) {
```

Comment:
Let's call the function latestVersion(currentVersion, newVersion) and return values 'current', 'new' and 'equal'

then you can make more natural sounding if statements.